src/modules/delete_todo.js

```diff
@@ -14,9 +14,8 @@ class deleteTodo {
     static removeFromLocalStorage = (index) => {
       const todos = getTodos.allTodos();
       const newTodo = todos.filter((td) => td.index !== Number(index));
-      // console.log(newTodo);
       // updateTodo.refreshId(newTodo);
-      addTodo.storeTodo(JSON.stringify(newTodo));
+      AddTodo.storeTodo(JSON.stringify(newTodo));
```

Comment:
[Optional] Kindly remove all commented code for a cleaner code 

Please check for other occurrences before merfing 👍